pitch.md

```diff
@@ -8,16 +8,101 @@ _Who are you helping, with your language? What is that person's
+the musicians want to focus on. This is an especially motivating factor because
+sight reading is honest to goodness first time interactions with a piece of music,
+and one cannot simply just look at music to find what they are looking for. Otherwise,
+it isn't sight reading.
+
+I feel the above is important because current musicians only look for the difficulty
+of a piece for sight reading. It would be cool for musicians to practice smart when
+practicing sight reading.
+
+For reference, here are things off the top of my head that contributes to music's
+complexity in a classical and jazz perspective (feel free to skip over; I just
+wanted to make a point):
+
+There are many chord progressions. Different kinds of cadences include:
+
+* Authentic (Perfect, Imperfect)
+* Plagal (Phrygian, Lydian etc.)
+* Half
+* Deceptive
+
+There are numerous kinds of chords. The obvious triads are major, minor, diminished,
+and augmented. 7th chords include major, minor, dominant, half-diminished, fully-
+diminished, and major-minor. Jazz musicians also add alterations, which include
+the b9, #9, #11/b5, and #5/b13.
+
+The different kinds of chords also lead to many more chord progressions than the ones
+just listed above. Inversions of chords also multiply the number of chords to consider.
+
+
+There are 12 major scales, one for each key. For each major scale, there are 7
+modes:
+
+* Ionian (the major scale itself)
+* Dorian
+* Phrygian
+* Lydian
+* Mixolydian
+* Aeolian (the natural minor of the major scale)
+* Locrian
+
+which gives us 12 * 7 = 84 different kinds of scales. There are also 12 harmonic
+and melodic minor scales for each key. In addition, there are artifial scales
+(whole tone, whole-half diminished, half-whole diminished), pentatonic scales
+(major and minor), and other scales derived from other cultures. Scales can also
+be built by specifying a few key base notes and randomly selecting other notes.
+
+Sight reading is also not just about reading notes correctly. It's also about playing
+the correct notes correctly. The texture and dynamics are extremely important. There
+is also a plethora of Italian, French, German, and other words that describe how
+one is supposed to play music as a few symbols does not cut the infinite number of ways
+to play music. There are dictionaries of these terms, and most musicians don't most
+of these terms off-hand.
+
+Pianists read two lines when sight reading while violinists and others tend to read
+one line. So for pianists, how the lines interact with each other is also a factor
+when reading music. Many contemporary composers have use polytonality in compositions
+(meaning a pianist reads a different key for each line). 
+
+There are also different clefs:
+
+* Treble
+* Bass
+* Alto
+* Tenor
+
+and reading ledger lines far above or below clefs is difficult (as is reading music
+with a lot of accidentals such as sharps and flats).
+
+The list goes on...
```

Comment:
This sounds really great for learning. It sounds like you can define the elements you want to practice sight reading for and then it will generate something that will test those specific elements which is super impressive. I question if someone wanted to test a similar chord progression repeatedly if the output would be too repetitive or if it would be random enough to still be a challenge.